javatests/google/registry/rde/imports/XjcToDomainResourceConverterTest.java

```diff
@@ -120,6 +120,8 @@ public void testConvertDomainResource() throws Exception {
     assertThat(domain.getLastEppUpdateTime()).isNull();
     assertThat(domain.getAutorenewBillingEvent()).isNotNull();
     assertThat(domain.getAutorenewPollMessage()).isNotNull();
+    assertThat(domain.getAuthInfo()).isNotNull();
+    assertThat(domain.getAuthInfo().getPw().getValue()).hasLength(16);
```

Comment:
You can use InjectRule's setStaticField to change XjcToDomainResourceConverter's stringGenerator field to be a DeterministicStringGenerator.  It's a pattern we use all over Nomulus.  There'll only be some slight adjustments to get this to work -- the field's type will have to be changed to StringGenerator, you'll need to make it non-final and package-private, and consequently you should annotate it with @NonFinalForTesting.